Dockerfile

```diff
@@ -0,0 +1,54 @@
+# TODO: Once stable, this should all go into the base image
+#
+# Base image at https://github.com/wordpress-mobile/docker-gb-mobile-image
+FROM public.ecr.aws/automattic/gb-mobile-image as gutenberg-test-runner
+
+WORKDIR /app
+
+ENTRYPOINT ["/bin/bash", "--login", "-c"]
+
+# Ensure the Node and npm version are the ones expected by the project
+COPY .nvmrc .nvmrc.host
+RUN nvm install $(cat .nvmrc.host)
+# Notice the old version of npm.
+RUN npm install -g npm@6
+
+# Add CI toolkit
+#
+# Using HTTPS to clone because there's no SSH key in the image
+RUN git clone https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin /ci-toolkit
+# Add ci-toolkit to the PATH
+RUN echo 'export PATH="$PATH:/ci-toolkit/bin"' >> ~/.bashrc
+
+# Install AWS CLI
+RUN apt-get update && apt-get install -y unzip
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscli.zip" && \
+    unzip awscli.zip && \
+    ./aws/install
+
+# Install jq
+RUN apt-get update && apt-get install -y jq
+
+# TODO: Once stable, this should all go into the base image
+#
+# Base image at https://github.com/wordpress-mobile/docker-android-build-image
+FROM public.ecr.aws/automattic/android-build-image as android-editor-test-runner
+
+WORKDIR /app
+
+ENTRYPOINT ["/bin/bash", "--login", "-c"]
+
+# Add CI toolkit
+#
+# Using HTTPS to clone because there's no SSH key in the image
+RUN git clone https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin /ci-toolkit
+# Add ci-toolkit to the PATH
+RUN echo 'export PATH="$PATH:/ci-toolkit/bin"' >> ~/.bashrc
+
+# Install AWS CLI
+RUN apt-get update
+RUN apt-get install -y unzip
+RUN apt-get install -y curl
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscli.zip" && \
+    unzip awscli.zip && \
+    ./aws/install
```

Comment:
I cannot build this locally, I get:

```
➜ make ci_build_docker_android_editor_image
docker build \
		--target android-editor-test-runner \
		--tag android-editor-test-runner \
		.
[+] Building 11.1s (8/11)
 => [internal] load build definition from Dockerfile                                                                 0.0s
 => => transferring dockerfile: 1.78kB                                                                               0.0s
 => [internal] load .dockerignore                                                                                    0.0s
 => => transferring context: 2B                                                                                      0.0s
 => [internal] load metadata for public.ecr.aws/automattic/android-build-image:latest                                2.3s
 => [android-editor-test-runner 1/8] FROM public.ecr.aws/automattic/android-build-image@sha256:855b2868b3a30c2aa95c  0.0s
 => CACHED [android-editor-test-runner 2/8] WORKDIR /app                                                             0.0s
 => CACHED [android-editor-test-runner 3/8] RUN git clone https://github.com/Automattic/a8c-ci-toolkit-buildkite-pl  0.0s
 => CACHED [android-editor-test-runner 4/8] RUN echo 'export PATH="$PATH:/ci-toolkit/bin"' >> ~/.bashrc              0.0s
 => ERROR [android-editor-test-runner 5/8] RUN apt-get update                                                        8.7s
------
 > [android-editor-test-runner 5/8] RUN apt-get update:
#8 1.199 Get:1 http://security.debian.org/debian-security stable-security InRelease [48.0 kB]
#8 1.540 Get:2 http://deb.debian.org/debian stable InRelease [147 kB]
#8 2.356 Get:3 http://deb.debian.org/debian stable-updates InRelease [52.1 kB]
#8 6.069 Reading package lists...
#8 8.665 E: Repository 'http://security.debian.org/debian-security stable-security InRelease' changed its 'Codename' value from 'bullseye-security' to 'bookworm-security'
#8 8.665 E: Repository 'http://deb.debian.org/debian stable InRelease' changed its 'Codename' value from 'bullseye' to 'bookworm'
#8 8.665 E: Repository 'http://deb.debian.org/debian stable-updates InRelease' changed its 'Codename' value from 'bullseye-updates' to 'bookworm-updates'
------
executor failed running [/bin/sh -c apt-get update]: exit code: 100
make: *** [ci_build_docker_android_editor_image] Error 1
```

I tried [android-build-image](https://github.com/wordpress-mobile/docker-android-build-image/) locally and it works. Or rather, it goes past `RUN apt-get update`. I killed the `docker build` because it was taking ages to install the Android SDK.